src/preact/base-element.js

```diff
@@ -651,7 +667,10 @@ function shouldMutationBeRerendered(Ctor, m) {
     const props = Ctor['props'];
     for (const name in props) {
       const def = props[name];
-      if (m.attributeName == def.attr) {
+      if (
+        (def.attr && m.attributeName == def.attr) ||
+        (def.attrs && def.attrs.indexOf(m.attributeName) != -1)
```

Comment:
Nit: 
```suggestion
        (def.attrs && def.attrs.includes(m.attributeName))
```

Reply:
Didn't realize that we polyfilled `Array.includes`.